scripts/Upgrade-M365DataCollector.ps1

```diff
@@ -96,6 +107,12 @@ $ContianerName = "m365-datacollection-script"
 $RunbookScriptName = "$ScriptPrefix-$DataCollectorVersion.ps1"
 $RunbookName = "$ScriptPrefix-$DataCollectorVersion"
 $path = "./runbooks"
+$SharePointDomainURL = "https://$SharePointDomain"
```

Comment:
convert SharepointDomainURL to SharePointAdminCenterURL as SharePoint domain URL is misleading

Reply:
done